dataprofiler/profilers/profiler_options.py

```diff
@@ -1574,7 +1574,7 @@ def __init__(self, presets: str = None) -> None:
         :vartype unstructured_options: UnstructuredOptions
         :ivar presets: A pre-configured mapping of a string name to group of options:
             "complete", "data_types", "numeric_stats_disabled",
-            and "memory_optimization". Default: None
+            and "lower_memory_sketching". Default: None
```

Comment:
nice will need to update the doc for this as well

Reply:
+1 @lizlouise1335